pkg/io/video/broadcast_test.go

```diff
@@ -47,3 +48,21 @@ func TestBroadcast(t *testing.T) {
 		t.Fatal("Expected actual frame without copy to be the same with the original")
 	}
 }
+
+func TestBroadcastWithCopyOnReadError(t *testing.T) {
+	expectedError := fmt.Errorf("expected error")
```

Comment:
```suggestion
	expectedError := errors.New("expected error")
```

Reply:
Github Copilot seems to still have some room to improvements 😛